docs/TRANSLATIONS.md

```diff
@@ -1,115 +1,114 @@
 # Translations
 
-## Help us to review/proofread translations
+## Overview
 
-You can join our [GitHub translation teams](https://github.com/orgs/organicmaps/teams/translations/teams),
-so any contributor can tag all teams (or a specific language team) to get help with the review.
+The project consists of multiple components, each with its own translation files. Translations are managed via [Weblate][weblate], which integrates with Git, syncs files, offers a user-friendly interface, and automatically submits PRs with updates.
 
-Please respond in the relevant [GitHub discussion](https://github.com/orgs/organicmaps/discussions/8538), or let us know at hello@organicmaps.app
+## Components
 
-## Contribute translations directly
+| Component                          | Description                                | Translation Files                                                                                   |
+| ---------------------------------- | ------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| [Android][android_weblate]         | UI strings                                 | [android/app/src/main/res/values\*/strings.xml][android_git] ([en][android_git_en])                 |
+| [iOS][ios_weblate]                 | UI strings                                 | [iphone/Maps/LocalizedStrings/\*.lproj/Localizable.strings][ios_git] ([en][ios_git_en])             |
+| [iOS Plurals][ios_plurals_weblate] | UI strings (plurals)                       | [iphone/Maps/LocalizedStrings/\*.lproj/Localizable.stringsdict][ios_git] ([en][ios_plurals_git_en]) |
+| [iOS Plist][ios_plist_weblate]     | UI strings (system-level)                  | [iphone/Maps/LocalizedStrings/\*.lproj/InfoPlist.strings][ios_git] ([en][ios_plist_git_en])         |
+| [Sounds][sounds_weblate]           | Sound strings for navigation               | [data/sound-strings/\*.json][sounds_git] ([en][sounds_git_en])                                      |
```

Comment:
`Voice announcement strings for navigation directions (TTS)`
instead of `Sound strings for navigation`

and TTS instead of Sounds

Reply:
And what is TTS? :-)

Reply:
Anyway, updated.

Reply:
Text To Speech :)